### DISCUSSION TITLE
Can we get all the data at once from the API , rather paginating using next url link ?

### DISCUSSION THREAD

**Discussion by purnima110895** [freelawproject/courtlistener#4079] [Q&A] [answered]
Like the url - https://www.courtlistener.com/api/rest/v3/dockets/?date_modified__gte=2024-05-01T00:00:00Z&date_modified__lte=2024-05-01T23:59:59Z , it has around total 16k records, the first page gives only 20 records, then we have to paginate using next url to get more records, can we get all total records from the API url ?

**Comment by mlissner** [comment, marked as answer]
No, there's no way to get everything in a single response because our poor database would suffer terribly if we attempted such a thing. Sorry!
